content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -0,0 +1,178 @@
++++
+title = "Pipelines Standalone Deployment in GCP"
+description = "Instructions to deploy Kubeflow Pipelines standalone to a cluster"
+weight = 15
++++
+
+As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize
+manifests in GCP.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+## Deploying Kubeflow Pipelines standalone to an existing cluster
+
+1. Deploy latest version of Kubeflow Pipelines:
+
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+```
+
+2. Get the URL for the Kubeflow Pipelines UI :
```

Comment:
The list formatting is broken - see preview:
https://deploy-preview-1174--competent-brattain-de2d6d.netlify.com/docs/pipelines/standalone-deployment-gcp/

To fix the formatting, indent the code snippets by 4 characters.

Reply:
Fixed